ott/geometry/pointcloud.py

```diff
@@ -567,6 +572,17 @@ def tree_unflatten(cls, aux_data, children):
         x, y, cost_fn=cost_fn, src_mask=src_mask, tgt_mask=tgt_mask, **aux_data
     )
 
+  def _cosine_to_sqeucl(self) -> 'PointCloud':
```

Comment:
is there a use case where this was needed?

Reply:
ATM I just use it in `k-means` as a nice enhancement, otherwise no. Am fine also removing this, it's a bit niche.

Reply:
i am fine with this. This feels like a fairly simple preprocessing, but it's ok as it is.